src/NUnitEngine/nunit.engine/Runners/DirectTestRunner.cs

```diff
@@ -71,7 +71,22 @@ public override TestEngineResult Explore(TestFilter filter)
             var result = new TestEngineResult();
 
             foreach (IFrameworkDriver driver in _drivers)
-                result.Add(driver.Explore(filter.Text));
+            {
+                string driverResult;
+
+                try
+                {
+                    driverResult = driver.Explore(filter.Text);
+                }
+                catch (Exception ex)
+                {
+                    if (ex is NUnitEngineException)
+                        throw;
```

Comment:
This is a new(ish) framework feature, right? (I hadn't heard of it, looks handy!) I thought NUnit supported back to 2.0 and given this is in the engine it might also be the case. Or is that not the case since this is the 'client side' of NUnit? I guess my question is: what's the minimum supported framework version for the engine? The project is currently set to target 2.0